### ISSUE TITLE
Accessibility issues Blog List Read More Link - Level A: 2.5.3 Label in Name

### ISSUE THREAD

**Issue by natechadwick** [percussion/percussioncms#627]
**Describe the bug**
There is an accessibility issue when using the Read More feature of the Blog List. 

**Level A: 2.5.3 Label in Name**
* Visible label and accessible name do not match
* The accessible name of any interactive element should contain its visible text label.

**To Reproduce**
Create a blog with a blog list and multiple blog posts.  In the blog post use the insert read more link feature to generate Read More links for the first few sentences of text.

**Expected behavior**
When validating the published blog index / list page with an Accessibility checker (SiteImprove, WebAIM etc) the more links should not be flagged with the Label in Name issue. 

It looks like what is currently happening is that the link is being dynamically generated with a **title** and **aria-label** attribute both with the link text / title of the page being linked to, with the text of the **a** tag being the text configured in the More Link layout property of the Blog List.  For example:

`<a .. href="/blog/myblogpost.html" title="My Blog Post Title" aria-label="My Blog Post Title">Read more...</a>`

What should be there:

`<a .. href="/blog/myblogpost.html" title="My Blog Post Title" aria-label="Read More...">Read more...</a>`

or as it seems like the aria-label is redundant here:

`<a .. href="/blog/myblogpost.html" title="My Blog Post Title">Read more...</a>`
 
I think the last one would be preferred accessibility wise, as aria-label seems redundant, but it would need tested.

**Comment by natechadwick** [comment]
Related files relative from repository root:

a[class='perc-more-link']

- delivery/common/js/views/PercBlogListView.js
- delivery/common/js/views/PercMostReadBlogPostsView.js
- delivery/common/js/views/PercResultView.js
- modules/perc-tinymce/src/main/resources/META-INF/resources/sys_resources/tinymce/plugins/percmorelink (TinyMCE plugin) 
- system/cms/content/applications/sys_resources/ApplicationFiles/vm/sys_assembly.vm
